test/extended/prometheus/prometheus.go

```diff
@@ -88,6 +88,20 @@ var _ = g.Describe("[sig-instrumentation][Late] Alerts", func() {
 		e2e.Logf("Watchdog alert is firing")
 	})
 
+	g.It("should not have any alerts in pending state the entire cluster run", func() {
```

Comment:
Could you move it closer https://github.com/openshift/origin/blob/master/test/extended/prometheus/prometheus.go#L54 as those tests are very similar and need almost the same exclusion list?